service/src/main/java/org/cbioportal/service/util/MutationMapperUtils.java

```diff
@@ -90,12 +101,16 @@
                             } else {
                                 try {
                                     Gene site2Gene = geneService.getGene(site2GeneSymbol);
-                                    if (site2Gene != null) {
-                                        structuralVariant.setSite2EntrezGeneId(site2Gene.getGeneticEntityId());
-                                        structuralVariant.setSite2HugoSymbol(site2Gene.getHugoGeneSymbol());
-                                    }
+                                    structuralVariant.setSite2EntrezGeneId(site2Gene.getEntrezGeneId());
+                                    structuralVariant.setSite2HugoSymbol(site2Gene.getHugoGeneSymbol());
                                 } catch (Exception e) {
-                                    // Site2 gene is not set when gene symbol is not found in database
+                                    // Site2 gene is not set when gene symbol is not found in database. Check if it is an alias
+                                    List<Gene> aliasGenes = geneService.getAllGenes(null, site2GeneSymbol, "SUMMARY",
```

Comment:
Where is this `SUMMARY` coming from? Why isn't this a constant?

Reply:
`SUMMARY` constant is present in web layer. moving it to `service` layer introduces lot of changes. better to fix it in another pr